<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_OrphansBase</class>
 <widget class="QFrame" name="PropertyWidget_OrphansBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>113</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="FormWidget" name="keepLabelStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="label" stdset="0">
        <string/>
       </property>
       <property name="direction">
        <enum>FormWidget::LabelPosition::Left</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ScrSpinBox" name="keepLinesStart">
          <property name="toolTip">
           <string>Ensure that first lines of a paragraph won't end up separated from the rest (known as widow/orphan control)</string>
          </property>
          <property name="suffix">
           <string> lines</string>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="FormWidget" name="keepLabelEnd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="label" stdset="0">
        <string/>
       </property>
       <property name="direction">
        <enum>FormWidget::LabelPosition::Left</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ScrSpinBox" name="keepLinesEnd">
          <property name="toolTip">
           <string>Ensure that last lines of a paragraph won't end up separated from the rest (known as widow/orphan control)</string>
          </property>
          <property name="suffix">
           <string> lines</string>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Switch" name="keepTogether">
     <property name="toolTip">
      <string>If checked, ensures that the paragraph won't be split across multiple pages or columns</string>
     </property>
     <property name="text">
      <string>Do not split paragraph</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Switch" name="keepWithNext">
     <property name="toolTip">
      <string>If checked, automatically moves the paragraph to the next column or page if the next paragraph isn't on the same page or column</string>
     </property>
     <property name="text">
      <string>Keep with next paragraph</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/form_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>Switch</class>
   <extends>QCheckBox</extends>
   <header>ui/widgets/switch.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>keepLinesStart</tabstop>
  <tabstop>keepLinesEnd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
